From 93b03119da8674a3eddb7879857e9f17fa16fd2a Mon Sep 17 00:00:00 2001
From: violet-sippial <38817272+violet-sippial@users.noreply.github.com>
Date: Sat, 28 Apr 2018 20:29:13 +0200
Subject: [PATCH] [subset] Do not compare ttx progress output in the tests

Suppress progress messages of ttx. This avoids comparing this output to the reference file which lets the test fail.
---
 test/subset/run-tests.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/subset/run-tests.py b/test/subset/run-tests.py
index 0b119fee6..1cd1a19f6 100755
--- a/test/subset/run-tests.py
+++ b/test/subset/run-tests.py
@@ -80,6 +80,7 @@ def run_test(test, should_check_ots):
 def run_ttx(file):
 	print ("ttx %s" % file)
 	cli_args = ["ttx",
+		    "-q",
 		    "-o-",
 		    file]
 	return cmd(cli_args)
